ext/page_log/palm/palm.c

```diff
@@ -662,7 +673,28 @@ palm_handle_discard(WT_PAGE_LOG_HANDLE *plh, WT_SESSION *session, uint64_t page_
       discard_args->base_lsn, discard_args->backlink_checkpoint_id,
       discard_args->base_checkpoint_id);
 
+    /* We always write full pages for tombstones. */
+    bool is_delta = false;
+
+    /* Create the tombstone, we don't expect any other flag set. */
+    assert(discard_args->flags == WT_PAGE_LOG_TOMBSTONE);
```

Comment:
@ddanderson, would I create the tombstone here and feed it as the `buf` arg of `palm_kv_put_page`? I have currently set it to `NULL` but that can't be right since we need the tombstone to be written.

Reply:
Sorry, I still don't see how a zero sized buffer would correspond to a tombstone. Unless this will be specific to PALM? If PALM reads a key/value pair which has a value of size 0, will we consider it as a tombstone?

Reply:
I just read the comment below. So we could have a zero sized buffer, an internal flag for PALM which would be `WT_PAGE_LOG_TOMBSTONE` and that's how we would find out if a record has a tombstone.